src/server/modules/admin/admin-v1/index.ts

```diff
@@ -0,0 +1,29 @@
+import { StorableUrl } from '../../../repositories/types'
+
+export { AdminApiV1Controller } from './AdminApiV1Controller'
+
+type LongUrlProperty = {
+  longUrl: string
+}
+type UserIdProperty = {
+  userId: number
+}
+
+type ShortUrlProperty = {
+  shortUrl: string
+}
+
+type EmailProperty = {
+  email: string
+}
+
+type ShortUrlOperationProperty = UserIdProperty & ShortUrlProperty
+
+export type UrlCreationRequest = ShortUrlOperationProperty &
+  LongUrlProperty &
+  EmailProperty
+
+export type UrlV1DTO = Pick<
+  StorableUrl,
+  'shortUrl' | 'longUrl' | 'state' | 'clicks' | 'createdAt' | 'updatedAt'
+>
```

Comment:
this appears to be unused as the admin API reuses the same `UrlV1Mapper` and hence `UrlV1DTO` from the external API v1 -- if we're okay to go ahead with that approach, would it be safe to remove this?